hubblestack/extmods/modules/pulsar.py

```diff
@@ -42,6 +43,9 @@
     DEFAULT_MASK = None
 
 __virtualname__ = 'pulsar'
+CONFIG = None
+CONFIG_STALENESS = 0
```

Comment:
Any reason you added these back in? I removed them because they appeared unused. Possible you just need a rebase.

Reply:
hrm, I think that's a merge artifact... checking.